### PR TITLE
[RW-9409][risk=no] Pause and resume RStudio GKE apps

### PR DESCRIPTION
**Pull request by Peter-Lavigne**
Description:

Allows users to pause and resume RStudio GKE apps.

Status: Running
image

Status: Pausing
image

Status: Paused
image

Status: Resuming
image

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have added explanatory comments where the logic is not obvious
- [x] I have run and tested this change locally, and my testing process is described here
- [x] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [x] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [x] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [x] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [x] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

**Comment by Peter-Lavigne** [comment]
@jmthibault79 I wasn't sure how thoroughly to test the pause / resume button. I was considering validating it for each app status but I wasn't sure how to do that in a way that works well across all apps. Also, I didn't know if you were currently testing that behavior and didn't want to overlap work.

**Review by jmthibault79 — APPROVED** [review]
Looks great, tested locally.

Note: I reran the e2e tests because it looked like they had many failures from yesterday's outage

File: ui/src/app/components/apps-panel/apps-panel-button.tsx [hunk 1]
```diff
@@ -56,7 +56,11 @@ interface Props {
 export const AppsPanelButton = (props: Props) => {
   const { disabled, onClick, icon, buttonText } = props;
   return (
-    <Clickable {...{ disabled, onClick }} style={{ padding: '0.5em' }}>
+    <Clickable
+      {...{ disabled, onClick }}
+      style={{ padding: '0.5em' }}
+      data-test-id={`apps-panel-button-${buttonText}`}
```
[the remark this answers is not in the document]
Comment (jmthibault79) [reply ui/src/app/components/apps-panel/apps-panel-button.tsx:62]: (I will also explore this in my testing PR)

[the remark this answers is not in the document]
Comment (jmthibault79) [reply ui/src/app/components/apps-panel/apps-panel-button.tsx:62]: I did not find a real use case for changing this.

**Pull request merged**